<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icon.jpg</normaloff>images/icon.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QSpinBox" name="sb_min">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>21</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_ret">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l_min">
     <property name="text">
      <string>最大间隔分钟数</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cb_del_tip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="l_del_tip">
     <property name="text">
      <string>删除确认</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_sure">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="l_new_autorun">
     <property name="text">
      <string>新建时自动启动</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cb_new_autorun">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
